File: solo.backend/Solo.Domain/Entities/Ticket.cs
namespace Solo.Domain.Entities
{
    public class Ticket : EntityBase
    {
        public int ParkObjectId { get; set; }
        public ParkObject ParkObject { get; set; }

        public int CustomerId { get; set; }
        public User Customer { get; set; }

        public bool Closed { get; set; }
        public TicketType Type { get; set; }
    }

    public enum TicketType
    {
        Adult,
        Child,
        Privileged
    }
}

File: solo.mobile/api/Ticket.ts
export interface Ticket {
    id: number;
    closed: boolean;
    customerId: number;
    type: TicketType;
    parkObjectId: number;
    queueNumber: number;
}

export enum TicketType {
    Adult = 0,
    Child = 1,
    Privileged = 2,
}
